src/main/java/de/muenchen/isi/api/validation/DistributionValidator.java

```diff
@@ -36,74 +50,68 @@ public boolean isWohneinheitenDistributionValid(final List<BauabschnittDto> baua
             bauratenFromAllTechnicalBaugebiete
         );
 
-        final var wohneinheiten = ObjectUtils.isEmpty(weGesamt) ? 0 : weGesamt;
-
         if (containsNonTechnicalBaugebiet) {
             final var sumVerteilteWohneinheitenBaugebiete = nonTechnicalBaugebiete
                 .stream()
-                .map(baugebiet -> ObjectUtils.isEmpty(baugebiet.getWeGeplant()) ? 0 : baugebiet.getWeGeplant())
+                .map(baugebiet -> {
```

Comment:
Evtl kurz über dem If-Else-Statements oder in den Javadocs der Methoden `isWeGfDistributionValid` und `isRealisierungVonDistributionValid` in den beiden Methoden beschreiben weshalb zwischen einem technischen und nichttechnischen Baugebiet unterschieden wird.

Ich musste erst mal überlegen weshalb hier unteschieden wird. Wenn der Code längere Zeit nicht mehr angeschaut wird, ist der Grund für die Unterscheidung nicht mehr ganz klar.